inst/script.R

```diff
@@ -19,6 +19,11 @@ touchstone::benchmark_run_ref(
   n = 6
 )
 
+# You can run tests only if the relevant file changed
+touchstone::run_on_change("R/utils.R", touchstone::benchmark_run_ref(
+  random_test = yourpkg::f(), #<- TODO put the call you want to benchmark here
+  n = 2
+))
 
 # create artifacts used downstream in the GitHub Action
 touchstone::benchmarks_analyze()
```

Comment:
I note this in a few files. Can you please use pre-commit on your prposed changes, there is an end-of-file-fixer that ensures you end with an empty line.

Reply:
Ah this is "your fault" ;P as I use r-lsp for formatting in vsc and they use ` styler::tidyverse_style(indent_by = options$tabSize)` 

But I will look into using precommit :)

Reply:
aww yes. Please upvote https://github.com/REditorSupport/languageserver/issues/462.